src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        UserService us = new UserService();
```

Comment:
let's make your naming more clear:
```suggestion
        UserService userService = new UserService();
```